### DISCUSSION TITLE
Skip checkup failure

### DISCUSSION THREAD

**Discussion by neilmacalisang** [skiptools/skip#298] [Q&A]
Hi, I'd just like to ask for help. I'm getting this error when running skip checkup and I'm not sure on the next steps I need to do to fix it.

Device info:
<img width="392" alt="Screenshot 2024-12-12 at 1 53 33 PM" src="https://github.com/user-attachments/assets/2ec78453-7963-4e25-88bc-91cce06b4cbb" />

executing command: /usr/local/bin/skip version
Skip version 1.2.7
[✓] Skip version 1.2.7 (= 1.2.7)
executing command: /usr/bin/sw_vers --productVersion
13.7.2
[✓] macOS version 13.7.2 (> 13.5.0)
executing command: /usr/sbin/sysctl -n sysctl.proc_translated
sysctl: unknown oid 'sysctl.proc_translated'
[!] Error running sysctl (Intel)
[✗] Command failed with exit code 1
[✗] Skip 1.2.7 checkup failed with 1 error and 1 warning
See https://skip.tools/docs/faq and https://community.skip.tools for help

**Comment by marcprux** [comment]
Can you try running `skip checkup --no-fail-fast` and let us know the output?

**Comment by neilmacalisang** [comment]
Hi here is the output after running it:

[✓] Skip version 1.2.7 (= 1.2.7)
[✓] macOS version 13.7.2 (> 13.5.0)
[!] Error running sysctl (Intel)
[✓] Swift version 5.9.2 (> 5.9.0)
[✓] Xcode version 15.2 (> 15.0.0)
[✓] Xcode tools SDKs: 6
[✓] Homebrew version 4.4.11 (> 4.1.0)
[✓] Gradle version 8.11.1 (> 8.6.0)
[✓] Java version 23.0.1 (> 21.0.0)
[✓] Android Debug Bridge version 1.0.41 (> 1.0.40)
[✓] Android Studio version: 4.1
[✓] Android tools SDKs: 4
[✓] Resolve dependencies (297.62s)
[✓] Build hello-skip (208.73s)
[✗] Test Swift (27.86s)
[✗] Test Kotlin (1.03s)
[✗] Archive iOS ipa (60.78s)
[✗] The file “Payload” doesn’t exist.
[✗] Skip 1.2.7 checkup failed with 4 errors and 1 warning
See https://skip.tools/docs/faq and https://community.skip.tools for help
See output log for error details: /tmp/skip-checkup-2024-12-12T23:21:32Z.txt
Error: 5 errors

And here is the content of the output log:
[error_log.txt](https://github.com/user-attachments/files/18118757/error_log.txt)

**Comment by marcprux** [comment]
The problem is that `skip checkup` is trying to build an app for iOS 17.2, but it isn't installed in your Xcode version. Can you launch Xcode and try installing a iOS 17.2 simulator?

**Comment by neilmacalisang** [comment]
Hi, there was progress after installing an iOS 17.2 simulator in Xcode. However there were still errors during checkup.

[✓] Skip version 1.2.7 (= 1.2.7)
[✓] macOS version 13.7.2 (> 13.5.0)
[!] Error running sysctl (Intel)
[✓] Swift version 5.9.2 (> 5.9.0)
[✓] Xcode version 15.2 (> 15.0.0)
[✓] Xcode tools SDKs: 6
[✓] Homebrew version 4.4.11 (> 4.1.0)
[✓] Gradle version 8.11.1 (> 8.6.0)
[✓] Java version 23.0.1 (> 21.0.0)
[✓] Android Debug Bridge version 1.0.41 (> 1.0.40)
[✓] Android Studio version: 4.1
[✓] Android tools SDKs: 4
[✓] Resolve dependencies (67.54s)
[⠄] Build hello-skip: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-frontend -frontend -c -filelist /var/folders/b4/n7t1yqb14818pxxylh68cv25…ate/var/folders/b4/n7t1yqb14818pxxylh68cv25ckmz87/T/1707648A-255C-46F2-8660-25A1AD2AE566/hello-skip/.build/x86_64-apple-macosx/debug/index/store -index-system-modules^R[C^[[A^[[B^[[B
[⠀] Build hello-skip: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-frontend -frontend -c -filelist /var/folders/b4/n7t1yqb14818pxxylh68cv25…ate/var/folders/b4/n7t1yqb14818pxxylh68cv25ckmz87/T/1707648A-255C-46F2-8660-25A1AD2AE566/hello-skip/.build/x86_64-apple-macosx/debug/index/store -index-system-modules^R
[⡀] Build hello-skip: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-frontend -frontend -c -filelist /var/folders/b4/n7t1yqb14818pxxylh68cv25…ate/var/folders/b4/n7t1yqb14818pxxylh68cv25ckmz87/T/1707648A-255C-46F2-8660-25A1AD2AE566/hello-skip/.build/x86_64-apple-macosx/debug/index/store -index-system-modules^R
[✓] Build hello-skip (90.59s)
[✗] Test Swift (91.79s)
[✗] Test Kotlin (1.17s)
[✓] Archive iOS ipa (96.92s)
[✓] Assemble HelloSkip-release.ipa 63 KB
[✓] Verify HelloSkip-release.ipa 63 KB
[✗] Assembling Android apk (109.33s)
[✗] Verify HelloSkip-release.apk: The file “app-release-unsigned.apk” couldn’t be opened because there is no such file.
[✓] Check Swift Package (0.83s)
[✓] Check Skip Updates: 1.2.7
[✗] Skip 1.2.7 checkup failed with 4 errors and 1 warning
See https://skip.tools/docs/faq and https://community.skip.tools for help
See output log for error details: /tmp/skip-checkup-2024-12-13T02:13:38Z.txt
Error: 5 errors

[skip-checkup-2024-12-13T02:13:38Z.txt](https://github.com/user-attachments/files/18121077/skip-checkup-2024-12-13T02.13.38Z.txt)
